docs/pages/access-controls/reference.mdx

```diff
@@ -334,10 +334,19 @@ who created the session.
 
 <Details title="Version Warning: Before 8.1" opened={false}>
 Teleport versions prior to 8.1 don't support the roles exemplified below.
-You may to create such roles after an upgrade, but in the event of a cluster
+You may create such roles after upgrading, but in the event of a cluster
```

Comment:
```suggestion
You may create these roles after upgrading, but in the event of a cluster
```
Otherwise it implies that some roles not shown here are invalid.

Reply:
That is what it implies. Anything that uses the new where clause for `session` or `ssh_session` won't work prior to 8.1.